### PR TITLE
:heavy_plus_sign: add R to picard 2.18.9 docker

### PR DESCRIPTION
**Pull request by sickler-alex**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Add R to the picard 2.18.9 docker. This is the latest version that we have dockers for and making this change should allow all of the picard tools in the alignment workflow to use the same version of picard. This would also mean that we don't need to maintain two versions of picard dockers; one with R one without.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [X] I have committed any related changes to the PR

### CODE REVIEW

File: picard/2.18.9/Dockerfile
Comment (dmiller15) [inline picard/2.18.9/Dockerfile:2]: Should also leave Miguel's maintainer here.

**Review by dmiller15 — CHANGES_REQUESTED** [review]
I'd prefer some tests on existing tools that rely on this docker to make sure it doesn't break anything and also confirmation that this r-base install will work for you.

**Comment by migbro** [comment]
Actually, as silly as it may sound, perhaps make this `picard:2.18.9R` (so create and place in `2.18.9R`, update config). Then once PR is merged, run a couple small test to make sure you get the same results, update the workflows to have the same pull, and we'll be good.

**Review by migbro — CHANGES_REQUESTED** [review]
Yeah, I'd actually change the dir version to 2.18.9R, update the config. Once merged and built, we can test it out before commenting this new docker to the workflows.

**Review by migbro — APPROVED** [review]
Good call on adding in that script. I forgot about that!

**Review by dmiller15 — APPROVED** [review]

**Pull request merged**
